### ISSUE TITLE
DISABLED test_rpc_profiling_remote_record_function (__main__. ProcessGroupRpcTestWithSpawn)

### ISSUE THREAD

**Issue by rohan-varma** [pytorch/pytorch#45145]
https://app.circleci.com/pipelines/github/pytorch/pytorch/217237/workflows/9eaaf92d-91de-4eae-8fe1-2dbc24484c75/jobs/7656484

Related to https://github.com/pytorch/pytorch/issues/45067

cc @ezyang @gchanan @zou3519 @bdhirsh @jbschlosser @anjali411 @pietern @mrshenli @pritamdamania87 @zhaojuanmao @satgera @gqchen @aazzolini @rohan-varma @jjlilley @osalpekar @jiayisuse @mrzzd @agolynski @SciPioneer @H-Huang @cbalioglu @xush6528

**Comment by rohan-varma** [comment]
It looks like the fundamental issue is not RPC related, but a larger issue w/the profiler: https://github.com/pytorch/pytorch/issues/45160

**Comment by mrshenli** [comment]
closed by #45162

**Comment by mrshenli** [comment]
cc @rohan-varma 

Reopen as this occurs again: https://app.circleci.com/pipelines/github/pytorch/pytorch/260715/workflows/948c9235-8844-4747-b40d-c14ed33f8dbb/jobs/10199987/steps

```
Jan 16 00:31:07   test_rpc_profiling_remote_record_function (__main__.ProcessGroupRpcTestWithSpawn) ... RPC was initialized with the PROCESS_GROUP backend which is deprecated and slated to be removed and superseded by the TENSORPIPE backend. It is recommended to migrate to the TENSORPIPE backend.
Jan 16 00:31:07 RPC was initialized with the PROCESS_GROUP backend which is deprecated and slated to be removed and superseded by the TENSORPIPE backend. It is recommended to migrate to the TENSORPIPE backend.
Jan 16 00:31:07 RPC was initialized with the PROCESS_GROUP backend which is deprecated and slated to be removed and superseded by the TENSORPIPE backend. It is recommended to migrate to the TENSORPIPE backend.
Jan 16 00:31:07 RPC was initialized with the PROCESS_GROUP backend which is deprecated and slated to be removed and superseded by the TENSORPIPE backend. It is recommended to migrate to the TENSORPIPE backend.
Jan 16 00:31:07 ERROR:root:Caught exception: 
Jan 16 00:31:07 Traceback (most recent call last):
Jan 16 00:31:07   File "/opt/conda/lib/python3.6/site-packages/torch/testing/_internal/common_distributed.py", line 303, in wrapper
Jan 16 00:31:07     fn()
Jan 16 00:31:07   File "/opt/conda/lib/python3.6/site-packages/torch/testing/_internal/dist_utils.py", line 90, in new_test_method
Jan 16 00:31:07     return_value = old_test_method(self, *arg, **kwargs)
Jan 16 00:31:07   File "/opt/conda/lib/python3.6/site-packages/torch/testing/_internal/distributed/rpc/rpc_test.py", line 1496, in test_rpc_profiling_remote_record_function
Jan 16 00:31:07     f"{local_name} not in {record_function_children_names}",
Jan 16 00:31:07   File "/opt/conda/lib/python3.6/unittest/case.py", line 682, in assertTrue
Jan 16 00:31:07     raise self.failureException(msg)
Jan 16 00:31:07 AssertionError: False is not true : aten::sigmoid not in ['aten::empty', 'aten::zero_', 'aten::zeros', 'aten::empty', 'aten::ones', 'aten::ones', 'aten::empty', 'aten::fill_', 'aten::add', 'aten::empty', 'aten::fill_']
Jan 16 00:31:07 exiting process with exit code: 10
Jan 16 00:31:07 Process 1 terminated with exit code 10, terminating remaining processes.
Jan 16 00:31:07 ERROR (1.230s)
```

**Comment by rohan-varma** [comment]
Fixed by #57517)
